app/app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+
+const app = express();
+
+// const handleErrors = require('errorhandler.js')
+
+const {getTopics} = require('./controllers/topics.controller')
+
+
+// app.use(handleErrors)
+
+
+app.get('/api/topics',getTopics)
+
+module.exports = app;
```

Comment:
This is a nice and clean start to the app.js file, don't forget that error handlers would typically go after the endpoints when you do uncomment that line